metrics-collectd/src/test/java/com/codahale/metrics/collectd/CollectdReporterTest.java

```diff
@@ -0,0 +1,159 @@
+package com.codahale.metrics.collectd;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import org.collectd.api.ValueList;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static com.codahale.metrics.collectd.ValueListAssert.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class CollectdReporterTest {
+
+    @ClassRule
+    public static Receiver receiver = new Receiver(25826);
+
+    private MetricRegistry registry;
+    private CollectdReporter reporter;
+
+    @Before
+    public void setUp() {
+        registry = new MetricRegistry();
+        reporter = CollectdReporter.forRegistry(registry).build(new Collectd("localhost", 25826));
+    }
+
+    @Test
+    public void handlesCounter() throws Exception {
+        Counter counter = registry.counter("api.rest.requests");
+        counter.inc(42);
+
+        reporter.report();
+
+        ValueList values = receiver.next();
+        assertNotNull(values.getHost());
+        assertEquals("api", values.getPlugin());
```

Comment:
AssertJ assertions are more generally used in the project than the JUnit one. We use `AssertJ` for verifying `ValueList`, so for the consitency reasons it would be nice to use them in all assertions.